hs_core/hydroshare/discovery.py

```diff
@@ -1,6 +1,7 @@
 # Resource discovery
 
 
+# Pabitra: Can I delete all these methods and this file?
```

Comment:
@hyi  I am just asking the question (comment) if I can delete this file as this file has no working code.